crossdock-go/call.go

```diff
@@ -48,22 +48,18 @@ func Call(t *testing.T, clientURL string, behavior string, args url.Values) {
 	require.NoError(t, err, "request %v failed", args)
 	defer res.Body.Close()
 
-	var results Results
+	var results []Entry
 	require.NoError(t, json.NewDecoder(res.Body).Decode(&results),
 		"failed to decode response for %v", args)
 
 	for _, result := range results {
-		if result.Status != "passed" && result.Status != "skipped" {
-			t.Errorf("request %v failed: %s", args, result.Output)
-		}
-	}
-}
+		if result.Status() != Passed && result.Status() != Skipped {
+			output := result.Output()
 
-// Results represents the response of an entire test
-type Results []Result
+			delete(result, statusKey)
+			delete(result, outputKey)
```

Comment:
It'll add noise to the output. This information is already available. The status is "failed" because that's the only time we print this message, and the output is printed separately in front of the tags.